.gitattributes

```diff
@@ -0,0 +1,5 @@
+# Treat all files in the Go repo as binary, with no git magic updating
+# line endings. Windows users contributing to Go will need to use a
+# modern version of git and editors capable of LF line endings.
+
+* -text
```

Comment:
Do we need this in other repos too?

Reply:
CRLF was causing problems when checking goimports on the new Travis CI windows workers. So probably we can add it as needed to the other repos.